Azure Databricks Level-400 Workshop content

mabalija opened the pull request:
Added new content for the Azure Databricks workshop. This gives the instructor the flow to execute the workshop.

benc-uk commented:
Mahesh could you take a look at a few things
- Perhaps have the Databricks logo as a header image, for an example of this have a look at [my Kubernetes lab](https://raw.githubusercontent.com/azurecitadel/azurecitadel.github.io/master/cloud-native/kubernetes.md) and look at the overlay_image part. You should move the logo to the /images/header directory if you do this
- **Datbricks** is a misspelling on the page a couple of times
- The sentence "This session follows by labs" doesn't make sense
- The part at the very bottom isn't formatted correctly, is it a table? I can't read or understand it

mabalija commented:
Ben,

I have updated misspelling, removed sentence and yes it is a table, but for some reason in my preview screen it shows nicely as a table in VS code but is not aligned properly in the citadel portal. Also modified the logo as per your recommendation. Could you please let me know if anything I can change for both logo and the table of links? or shall I remove it altogether for now?

mabalija commented:
Ben, 

Rich helped me to fix the issues. Could you please merge to main branch.

Thanks,
Mahesh.B.
